### PR TITLE
Netstack image collection bugfix

### PR DESCRIPTION
**Pull request by internet-diglett**
I made a bit of an oopsie when I updated the netstack logic to use the environment variable for the target zpool used for image extraction. <s>I fixed that, and I also updated the README with the `netrc` information, as promised to @andrewjstone.</s>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,6 +18,17 @@ Install `propolis-server` from the
 The`get-propolis.sh` script can also be used to automatically install
 propolis-server form the current falcon CI build.
 
+To download certain resources you wil need to create a GitHub
```
Comment (rcgoodfellow) [inline README.md:21]: I think we can remove this bit now that the resources are public.

Comment (internet-diglett) [reply README.md:21]: Will do, I just want to do some testing to make sure that we've opened all of the repos that require it.

Comment (internet-diglett) [reply README.md]: Looks like we're good on this PR

**Review by rcgoodfellow — APPROVED** [review]
Thanks!

**Pull request merged**